translations/messages.en.yaml

```diff
@@ -24,6 +25,7 @@ commented: Commented
 change_view: Change view
 filter_by_time: Filter by time
 filter_by_type: Filter by type
+filter_by_subscription: Filter by subscription
```

Comment:
this appears unused, unless git is hiding it from me, maybe it was meant to be `filter_by_federation`?

Reply:
Hm? It's used for mouseover text, I think.

```
$ grep -r filter_by_subscription `ls | egrep -v var\|public`
templates/entry/_options.html.twig:                        aria-label="{{ 'filter_by_subscription'|trans }}"
templates/entry/_options.html.twig:                        title="{{ 'filter_by_subscription'|trans }}">
translations/messages.en.yaml:filter_by_subscription: Filter by subscription
$
```